packages/core/src/colors.ts

```diff
@@ -138,6 +138,13 @@ const colors = {
     grayDark: '#e4e4e4',
     frontpageArticle: '#f7fafd',
   },
+
+  /**
+   * Icon colors
+   * */
+  icon: {
+    iconBlue: '#4E51F2',
```

Comment:
Pirk. Men alt anna bruker små bokstaver.
```suggestion
    iconBlue: '#4e51f2',
```